sdk/rust/oak/src/http/mod.rs

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Functionality to help Oak Nodes create HTTP server pseudo-Nodes.
+use crate::{
+    invocations::{Invocation, ServerInvocationChannel},
+    io::{Receiver, ReceiverExt, SenderExt},
+    OakStatus,
+};
+
+use oak_abi::proto::oak::encap::{HttpRequest, HttpResponse};
+
+pub type HttpInvocation = Invocation<HttpRequest, HttpResponse>;
```

Comment:
Maybe just call this `Invocation`, so that it can be referred to as `oak::http::Invocation`? And similarly for the gRPC one.

Reply:
edit: in fact it seems that's already the case for the gRPC one, so this should probably just follow the same pattern.